grails-app/services/au/org/ala/profile/hub/ExportService.groovy

```diff
@@ -126,25 +123,140 @@ class ExportService {
             withPool(THREAD_COUNT) {
                 children.eachParallel {
                     if (it.profileId != params.profileId && it.scientificName != params.profileId) {
-                        model.profiles << loadProfileData(it.profileId, model.opus, params)
+                        curatedModel.profiles << loadProfileData(it.profileId, opus, params, latest)
                     }
                 }
             }
 
-            model.profiles = model.profiles.sort { it.profile.scientificName }
+            curatedModel.profiles = curatedModel.profiles.sort { it.profile.scientificName }
         }
 
-        wkhtmltoxService.makePdf(
-                view: "/pdf/_profile",
-//                footer: "/pdf/_pdfFooter",
-//                header: "/pdf/_pdfHeader",
-                model: model,
-                marginLeft: 15,
-                marginTop: 20,
-                marginBottom: 20,
-                marginRight: 15,
-                headerSpacing: 10,
-        )
+        curatedModel.options << [
+                allowFineGrainedAttribution: opus.allowFineGrainedAttribution,
+                displayToc: curatedModel.profiles?.size() > 1
+        ]
+
+        curatedModel << [
+                cover: [
+                        title       : opus.title,
+                        subtitle    : curatedModel.profiles[0]?.profile?.fullName,
```

Comment:
Might need to test this with a profile that does not have a matched name...I can't remember if the fullName is always populated. If it isn't, then use fullName ?: scientificName.